tools/cli/commands/changelog.js

```diff
@@ -817,61 +806,64 @@ async function promptChangelog( argv, needChangelog, types ) {
  * Prompts you for how you want changelogger to run (add to all projects or not, etc).
  *
  * @param {object} argv - the arguments passed.
- * @param {Array} needChangelog - files that need changelogs.
+ * @param {Array} defaultProjects - projects that use the default changelog types.
  * @param {Array} uniqueProjects - projects with unique changelog types.
  * @returns {argv}.
  */
-async function changelogAddPrompt( argv, needChangelog, uniqueProjects ) {
-	const totalProjects = [ ...needChangelog, ...uniqueProjects ];
+async function changelogAddPrompt( argv, defaultProjects, uniqueProjects ) {
+	const totalProjects = [ ...defaultProjects, ...uniqueProjects ];
+	let prompts;
 
-	// If only one project needs a changelog, confirm.
+	/* changelogConfirm: Add changelog for the one project? */
 	if ( totalProjects.length === 1 ) {
-		const response = await inquirer.prompt( {
+		prompts = {
 			type: 'confirm',
 			name: 'changelogConfirm',
 			message: `Add changelog for ${ totalProjects[ 0 ] }?`,
-		} );
-		return response;
-	}
-
-	// If multiple projects need a changelog that can be the same, confirm.
-	if ( uniqueProjects.length === 0 && needChangelog.length > 1 ) {
-		const response = await inquirer.prompt( [
+		};
+	} else if ( defaultProjects.length <= 1 && uniqueProjects.length > 0 ) {
+		/* separateChangelogFiles: Found N projects needing changelogs. Create for each one? */
+		prompts = {
+			type: 'confirm',
+			name: 'separateChangelogFiles',
+			message: `Found ${ totalProjects.length } project(s) that need a changelog. Create changelog for each one?`,
+		};
+	} else if ( defaultProjects.length > 0 && uniqueProjects.length === 0 ) {
+		console.log( "Where I'm supposed to be" );
+		/* sameChangelog: Found N projects. Same to all? */
+		prompts = [
 			{
 				type: 'confirm',
-				name: 'sameChangelogFiles',
-				message: `Found ${ needChangelog.length } project(s) that need a changelog. Create and add same changelog to all projects?`,
+				name: 'sameChangelog',
```

Comment:
Heh. I didn't really intend to rename this from "sameChangelogFiles" to "sameChangelog", I must have just left off the "Files". Either way works though.